src/networkLayer/client.js

```diff
@@ -0,0 +1,206 @@
+/* @flow */
+import Relay from 'react-relay/classic';
+import RequestObject from './requestObject';
+
+export type ClientNetworkLayerOpts = {
```

Comment:
To be more specific:
 - csrf
 - on403callback
 - onSpeedbarResponse
 - onJailedCallback
shouldn't be in relay-helpers

and onMutationStartCallback and onMutationProgressCallback don't appear to be used